src/libostree/ostree-sign-ed25519.c

```diff
@@ -546,6 +546,8 @@ _load_pk_from_file (OstreeSign *self,
         return glnx_throw (error, 
                            "signature: ed25519: no valid keys in file '%s'",
                            filename);
+      else
+        return FALSE;
```

Comment:
Well...there's a larger rework that should take place here.  But yes this is right for now.